src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <title>To do list:</title>
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <section class="main-container">
+        <section class="myToDoList">
+          <h1 class="head">
+            <span class="title"> Today's todo:</span>
+            <i class="fa-solid fa-repeat"></i>
+          </h1>
+          <div class="adding-to-do-list">
+            <span class="addList">
+              <input type="text" class="add-todo-input" id="toDoItem"
+            /></span>
+            <label for="add-todo"
+              ><i class="add-todo-icon fa-solid fa-right-from-bracket"></i
+            ></label>
+          </div>
+          <div>
+            <ul class="to-do-list"></ul>
+            <div class="button-container">
+              <button class="my-button"><p>clear all completed</p></button>
+            </div>
+          </div>
+        </section>
+      </section>
+    </main>
+    <footer></footer>
+  </body>
+  <script type="module" src="index.js"></script>
```

Comment:
As webpack is used in the project, the index.js script will be dynamically added to index.html by webpack. I think you don't have to link the script here.